automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -121,55 +121,77 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
+    verifyCluster(clusterTimeResult.result, googleProject, clusterName,
+      Seq(ClusterStatus.Creating), clusterRequest, bucketCheck)
 
     // verify with get()
     val creatingCluster = eventually {
-      clusterCheck(Leonardo.cluster.get(googleProject, clusterName), googleProject, clusterName, Seq(ClusterStatus.Creating), clusterRequest)
+//      verifyCluster(Leonardo.cluster.get(googleProject, clusterName), googleProject, clusterName,
+//        Seq(ClusterStatus.Creating), clusterRequest, bucketCheck)
+      verifyCluster(Leonardo.cluster.get(googleProject, clusterName), googleProject, clusterName,
+        Seq(ClusterStatus.Creating), clusterRequest)
     }(getAfterCreatePatience, implicitly[Position])
 
     if (monitor) {
       // wait for "Running", "Stopped", or error (fail fast)
       implicit val patienceConfig: PatienceConfig = clusterPatience
-      val expectedStatuses = if (clusterRequest.stopAfterCreation.getOrElse(false)) {
-        Seq(ClusterStatus.Stopped, ClusterStatus.Error)
-      } else {
-        Seq(ClusterStatus.Running, ClusterStatus.Error)
-      }
+
+      val expectedStatuses =
+        if (clusterRequest.stopAfterCreation.getOrElse(false)) {
+          Seq(ClusterStatus.Stopped, ClusterStatus.Error)
+        } else {
+          Seq(ClusterStatus.Running, ClusterStatus.Error)
+        }
+
       val runningOrErroredCluster = Try {
         eventually {
-          clusterCheck(Leonardo.cluster.get(googleProject, clusterName), googleProject, clusterName, expectedStatuses, clusterRequest)
+          verifyCluster(Leonardo.cluster.get(googleProject, clusterName), googleProject, clusterName,
+            expectedStatuses, clusterRequest)
         }
       }
 
       // Save the cluster init log file whether or not the cluster created successfully
       implicit val ec: ExecutionContextExecutor = ExecutionContext.global
-      saveDataprocLogFiles(creatingCluster).recover { case e =>
-        logger.error(s"Error occurred saving Dataproc log files for cluster ${creatingCluster.googleProject}/${creatingCluster.clusterName}", e)
-        None
-      }.futureValue match {
-        case Some((initLog, startupLog)) =>
+      saveDataprocLogFiles(creatingCluster).onComplete {
+        case Success(Some((initLog, startupLog))) =>
           logger.info(s"Saved Dataproc init log file for cluster ${creatingCluster.googleProject}/${creatingCluster.clusterName} to ${initLog.getAbsolutePath}")
           logger.info(s"Saved Dataproc startup log file for cluster ${creatingCluster.googleProject}/${creatingCluster.clusterName} to ${startupLog.getAbsolutePath}")
-        case None => logger.warn(s"Could not obtain Dataproc log files for cluster ${creatingCluster.googleProject}/${creatingCluster.clusterName}")
+        case Success(None) =>
+          logger.warn(s"Could not obtain Dataproc log files for cluster ${creatingCluster.googleProject}/${creatingCluster.clusterName}")
```

Comment:
Refactored  the code block to add the `Failure` `case` so if the log files can't be saved, the test fails. Previously, the test would still pass.